$NetBSD: patch-pyproject.toml,v 1.1 2023/08/09 13:19:19 adam Exp $

Do not install extra files as they end up in common directory.

--- pyproject.toml.orig	2023-08-09 13:14:09.000000000 +0000
+++ pyproject.toml
@@ -28,11 +28,6 @@ classifiers = [
     "Topic :: Software Development :: Build Tools",
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
-include = [
-    "CHANGELOG.md",
-    "MIGRATION.md",
-    "LICENSE.txt",
-]
 
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
